python/lsst/ts/m2/model.py

```diff
@@ -0,0 +1,849 @@
+
+        self.axial_forces = dict(
+            [
+                (item, np.zeros(self.n_actuators))
+                for item in (
+                    "lutGravity",
+                    "lutTemperature",
+                    "applied",
+                    "measured",
+                    "hardpointCorrection",
+                )
+            ]
+        )
+        self.tangent_forces = dict(
+            [
+                (item, np.zeros(self.n_tangent_actuators))
+                for item in (
+                    "lutGravity",
+                    "lutTemperature",
+                    "applied",
+                    "measured",
+                    "hardpointCorrection",
+                )
+            ]
+        )
+        self.force_balance = dict(
+            [(axis, 0) for axis in ("fx", "fy", "fz", "mx", "my", "mz")]
+        )
+        self.force_balance_system_status = False
+
+        self.lut = dict()
+
+        self.error_cleared = True
+
+    def _get_default_temperature(self, init_temperature=11.0):
+        """Get the default temperature. The unit is degree C.
+
+        Parameters
+        ----------
+        init_temperature : `float`, optional
+            Initial temperature in degree C. (default is 11.0)
+
+        Returns
+        -------
+        `dict`
+            Temperature sensor data. The key is sensor's position or reference.
+        """
+
+        temperature = dict()
+        temperature["ring"] = [init_temperature] * 12
+        temperature["intake"] = [init_temperature] * 2
+        temperature["exhaust"] = [init_temperature] * 2
+        temperature["ref"] = 21.0
+
+        return temperature
+
+    @property
+    def mirror_weight(self):
+        """Mirror weight (in Newton)."""
+        return 15578.0
+
+    @property
+    def max_axial_force(self):
+        """Axial force limit (in Newtons).
+
+        According with the vendor documents these limits are hard-coded in
+        the real system, so they are also hard-coded in the simulator.
+        """
+        return 444.82
+
+    @property
+    def max_tangent_force(self):
+        """Tangent force limit (in Newtons).
+
+        According with the vendor documents these limits are hard-coded in
+        the real system, so they are also hard-coded in the simulator.
+        """
+        return 4893.04
+
+    @property
+    def position_limit_radial(self):
+        """Mirror radial motion limit (in mm)."""
+        return 7.74
+
+    @property
+    def position_limit_z(self):
+        """Mirror motion limit in the optical direction (in mm)."""
+        return 7.89
+
+    def configure(self, config_dir, lut_path):
+        """Do the configuration.
+
+        Parameters
+        ----------
+        config_dir : `pathlib.PosixPath`
+            Configuration directory.
+        lut_path : `str`
+            Look-up table (LUT) path.
+        """
+
+        # Look-up table (LUT) factors (filename, basically).
+        lut_factors = set(
```

Comment:
You can create a set in python with `{}`, e.g;

```
lut_factors =  {
                 "F_0.csv",
                 "F_A.csv",
                 "F_E.csv",
                 "F_F.csv",
                 "Tr.csv",
                 "Tu.csv",
                 "Tx.csv",
                 "Ty.csv",
                 "temp_inv.txt",
             }
```

Reply:
Sure. Thanks for the suggestion!